client/lib/plans/index.js

```diff
@@ -262,3 +262,10 @@ export function applyTestFiltersToPlansList( planName ) {
 
 	return filteredPlanConstantObj;
 }
+
+export const isSkipPlansTestEnabled = () => {
+	return isEnabled( 'plans/skip-plans-for-free' ) &&
+		isEnabled( 'plans/personal-plan' ) &&
+		abtest( 'personalPlan' ) === 'show' &&
+		abtest( 'skipPlansLinkForFree' ) === 'skipPlansForFree';
```

Comment:
If a user already has the personalPlan variant set, does 100% force them to show? Otherwise folks might have `skipPlansLinkForFree` and `personalPlan` to hide which may make the results a bit more difficult to look at.